docs/sdk/auth/fragments/android/how-it-works.md

```diff
@@ -23,23 +23,31 @@ The AWSMobileClient manages your application session for authentication related
 
 This allows you to write workflows in your application based on the state of the user and what you would like to present on different screens. The `AWSMobileClient` also offers realtime notifications on user state changes which you can register for in your application using `.addUserStateListener` as in the code below.
 
-```swift
-AWSMobileClient.default().addUserStateListener(self) { (userState, info) in
-            switch (userState) {
-            case .guest:
-                print("user is in guest mode.")
-            case .signedOut:
-                print("user signed out")
-            case .signedIn:
-                print("user is signed in.")
-            case .signedOutUserPoolsTokenInvalid:
-                print("need to login again.")
-            case .signedOutFederatedTokensInvalid:
-                print("user logged in via federation, but currently needs new tokens")
+```java
+AWSMobileClient.getInstance().addUserStateListener(new UserStateListener() {
+    @Override
+    public void onUserStateChanged(UserStateDetails userStateDetails) {
+        switch (userStateDetails.getUserState()){
```

Comment:
[`WhitespaceAround` for `LCURLY`](https://checkstyle.sourceforge.io/config_whitespace.html#WhitespaceAround)

Reply:
fixed

Reply:
I still see the `){`